### PR TITLE
docs: New Using Pipes in templates doc

### PR DESCRIPTION
**Pull request by trekladyone**
New Using Pipes in templates doc

### CODE REVIEW

File: aio/content/guide/pipe-template.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Using a pipe in a template
```
Comment (dylhunn) [inline aio/content/guide/pipe-template.md:1]: Was this moved from elsewhere? If so, can we delete the old section and create the new page in the same PR?

**Review by dylhunn — CHANGES_REQUESTED** [review]

**Review by josephperrott — APPROVED** [review]
LGTM

Reviewed-for: code-ownership

File: aio/content/guide/pipe-template.md (hunk 1 above)
Comment (trekladyone) [reply aio/content/guide/pipe-template.md:1]: The original Pipes document was very long and has been broken down into separate Pipes documents.  This is derived from that original document with the help of Alex Rickabaugh.

**Comment by dylhunn** [comment]
Thanks, please dismiss my review as you already have an LGTM!

**Pull request closed without merging**